src/pages/Account/Login.tsx

```diff
@@ -1,17 +1,37 @@
+import React, { useState } from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { useSAMLSignIn } from "hooks/useSAMLSignIn";
+
 import LoginForm from "components/organisms/AuthenticationModal/LoginForm";
 import PasswordResetForm from "components/organisms/AuthenticationModal/PasswordResetForm";
 import RegisterForm from "components/organisms/AuthenticationModal/RegisterForm";
 import { InitialForm } from "components/organisms/AuthenticationModal/InitialForm";
-import React, { FC, useState } from "react";
+
+import SAMLLoginIcon from "assets/icons/saml-login-icon.png";
+
+// @debt move all styles into `Login.scss`;
 import "./Account.scss";
+import "./Login.scss";
 
-interface LoginProps {
+export interface LoginProps {
   formType?: "initial" | "login" | "register" | "passwordReset";
+  venue: AnyVenue;
 }
 
-export const Login: FC<LoginProps> = ({ formType = "initial" }) => {
+export const Login: React.FC<LoginProps> = ({
+  formType = "initial",
+  venue,
+}) => {
   const [formToDisplay, setFormToDisplay] = useState(formType);
 
+  const { signInWithSAML, hasSamlAuthProviderId } = useSAMLSignIn(
```

Comment:
You can remove the const at line 33 and convert it directly here
```suggestion
  const { signInWithSAML, hasSamlAuthProviderId: hasAlternativeLogins } = useSAMLSignIn(
```

Reply:
It's an explicit choice to have it both `hasSamlAuthProviderId` and `hasAlternativeLogins`.

The reason for that is that hasAlternativeLogins will have more things in it(Google auth, facebook auth, etc...)